_data/tables/transform-high-performing-org.yml

```diff
@@ -39,22 +44,60 @@
       data:
         list: false
         fr: 
-        en: Establish communication plan and change management for continuous improvement in (and around) IITB
+        en: Develop a self assessement tool based on capabilities of high performing organizations and PwC recommendations
     - col:
       data:
         list: false
         fr: 
-        en: 
+        en: To help teams understand the current condition and set measurable targets, the self assessment tool will guide teams toward ways to improve based on an evaluation of diferent capabilities linked to high performing organizations.
```

Comment:
Id change language from "to help" to 

Team managers set measureable targets for their teams to improve upon, which are tied to the performance improvements of their management (terminating at the CIO). 

Id vote for more definitive actions. "to help" is a bit hand wavey ... we could still help and they could still not properly define their metrics. I would just state "define their metrics" as the action

Reply:
That is the description.  the action here is : "Develop a self assessement tool based on capabilities of high performing organizations and PwC recommendations"